src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1919,6 +1930,46 @@ private boolean isRespondentDetailsConfidential(CaseData caseData) {
         }
     }
 
+    private boolean isApplicantRepresented(CaseData caseData) {
+
+        if (PrlAppsConstants.C100_CASE_TYPE.equals(CaseUtils.getCaseTypeOfApplication(caseData))) {
+            return isCaApplicantRepresented(caseData);
+        } else {
+            return isDaApplicantRepresented(caseData);
+        }
+    }
+
+    public boolean isCaApplicantRepresented(CaseData caseData) {
+        Optional<List<Element<PartyDetails>>> applicantsWrapped = ofNullable(caseData.getApplicants());
+
+        if (applicantsWrapped.isPresent() && !applicantsWrapped.get().isEmpty()) {
+            List<PartyDetails> applicants = applicantsWrapped.get()
+                .stream()
+                .map(Element::getValue).toList();
+
+            List<PartyDetails> unRepresentedApplicants = applicants.stream()
+                .filter(
+                    applicant -> (
+                        null != applicant.getUser().getSolicitorRepresented()
+                        && No.equals(applicant.getUser().getSolicitorRepresented())
+                    )
```

Comment:
this logic will not work unless there is NoC performed on the case.

You need to check this field - doTheyHaveLegalRepresentation first. On top of it, you must check representativeFirstName and representativeLastName is not empty

Reply:
doTheyHaveLegalRepresentation check impplemented along with solictiorEmail . So resolved.